.github/workflows/tests.yaml

```diff
@@ -195,11 +195,48 @@ jobs:
           sync
           sync
 
-  build:
+  integration_tests_rbs_volumes:
     runs-on: ubuntu-24.04
     needs:
-      - integration_tests_l2
       - integration_tests_lb_instances
+      - integration_tests_l2
+    concurrency: API
```

Comment:
They extended 'concurrency'

https://docs.github.com/en/actions/how-tos/write-workflows/choose-when-workflows-run/control-workflow-concurrency

We need a group name (I think, it should be 'rbs-specific' group name, because server reinstallation can run in parallel with RBS).

Also, `cancel-in-progress: false`, or add some 'if: always()' code to do cleanup.